doc/source/configuring.rst

```diff
@@ -51,7 +51,19 @@ include a password.
 
 If the password is not provided in the configuration file and ``use_keyring`` is
 not true, ``imapautofiler`` will prompt for a value when it tries to connect to
-the server.
+he server.
+
+You can also optionally provide the imap servers port and a custom CA file.
```

Comment:
Since you'll need to update this PR to make flake8 happy anyway, could you change "imap" to "IMAP" here, please?

Reply:
Yup, happy to change, also looks like I chopped off the t in the, I'll fix that as well. Other than flake8 are there any other linters you would like me to run this through?

Reply:
The tests run by travis are sufficient, thanks!